### ISSUE TITLE
Process has died, Unhandled exception

### ISSUE THREAD

**Issue by tkkhuu** [RIVeR-Lab/apriltags_ros#23]
Hi,

I trying to use the apriltag_ros package for a robotics project and I keep getting an error when running the example.launch file

`$ roslaunch apriltags_ros example.launch
... logging to /home/hayward_transbot/.ros/log/98ff82a4-72cb-11e7-a9f5-506583e696c6/roslaunch-arm-14974.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://192.168.137.5:39794/

SUMMARY

PARAMETERS
 * /apriltag_detector/image_transport: compressed
 * /apriltag_detector/projected_optics: True
 * /apriltag_detector/tag_descriptions: [{'id': 0, 'size'...
 * /apriltag_detector/tag_family: 36h11
 * /rosdistro: kinetic
 * /rosversion: 1.12.7

NODES
  /
    apriltag_detector (apriltags_ros/apriltag_detector_node)

ROS_MASTER_URI=http://192.168.137.5:11311/

core service [/rosout] found
process[apriltag_detector-1]: started with pid [14992]
[ INFO] [1501167511.554384042]: Loaded tag config: 0, size: 0.163513, frame_name: tag_0
[ INFO] [1501167511.567628783]: Loaded tag config: 1, size: 0.163513, frame_name: a_frame
[ INFO] [1501167511.574866464]: Loaded tag config: 2, size: 0.163513, frame_name: tag_2
[ INFO] [1501167511.582110436]: Loaded tag config: 3, size: 0.163513, frame_name: tag_3
[ INFO] [1501167511.590937514]: Loaded tag config: 4, size: 0.163513, frame_name: tag_4
[ INFO] [1501167511.597694922]: Loaded tag config: 5, size: 0.163513, frame_name: tag_5
[apriltag_detector-1] process has died [pid 14992, exit code -11, cmd /home/hayward_transbot/catkin_ws/devel/lib/apriltags_ros/apriltag_detector_node image_rect:=/webcam/image camera_info:=/webcam/camera_info __name:=apriltag_detector __log:=/home/hayward_transbot/.ros/log/98ff82a4-72cb-11e7-a9f5-506583e696c6/apriltag_detector-1.log].
log file: /home/hayward_transbot/.ros/log/98ff82a4-72cb-11e7-a9f5-506583e696c6/apriltag_detector-1*.log
all processes on machine have died, roslaunch will exit
shutting down processing monitor...
... shutting down processing monitor complete
done
Unhandled exception in thread started by 
sys.excepthook is missing
lost sys.stderr
$ `

Does anyone else have this same problem? I need help debugging this, if there's anything else I can provide to help please let me know.

Thanks,
Tri

**Comment by Wiesen** [comment]
Hi, @tkkhuu 

I encountered the same problem when I launch detector in Odroid XU4, but it runs well on normal PC. Do you have any idea how to solve it now?

Thanks
Yang

**Comment by tkkhuu** [comment]
Hi @Wiesen,

What version of ROS did you install on the Odroid XU4? I installed barebone ROS Kinetic on the BeagleBone. I think the bare bone installation might have missed some dependencies that the error log did not tell us. I had the same problem when I install bare bone on an old PC. So I ended up installing the desktop-full-version and it fixed the problem.

`sudo apt-get install ros-kinetic-desktop-full`

instead of 

`sudo apt-get install ros-kinetic-ros-base`

Tri
